pyproject.toml

```diff
@@ -31,6 +33,8 @@ dependencies = [
   "python-dotenv~=1.0",
   "typer[all]~=0.9",
   "pyyaml~=6.0",
+  'eval-type-backport; python_version<"3.10"',
```

Comment:
See #33 for why this library is needed (tl;dr so that pydantic can parse the `|` union notations on Python <3.10)